src/SamV71Core/SamV71Core.c

```diff
@@ -26,125 +26,96 @@
 #include <Pmc/Pmc.h>
 #include <Mpu/Mpu.h>
 
+#include <bsp/atsam-clock-config.h>
+
+#define CKGR_PLLAR_DIVA_Pos 0
+#define CKGR_PLLAR_DIVA_Msk (0xffu << CKGR_PLLAR_DIVA_Pos)
+#define CKGR_PLLAR_DIVA(value) ((CKGR_PLLAR_DIVA_Msk & ((value) << CKGR_PLLAR_DIVA_Pos)))
+#define CKGR_PLLAR_PLLACOUNT_Pos 8
+#define CKGR_PLLAR_PLLACOUNT_Msk (0x3fu << CKGR_PLLAR_PLLACOUNT_Pos)
+#define CKGR_PLLAR_PLLACOUNT(value) ((CKGR_PLLAR_PLLACOUNT_Msk & ((value) << CKGR_PLLAR_PLLACOUNT_Pos)))
+#define CKGR_PLLAR_MULA_Pos 16
+#define CKGR_PLLAR_MULA_Msk (0x7ffu << CKGR_PLLAR_MULA_Pos)
+#define CKGR_PLLAR_MULA(value) ((CKGR_PLLAR_MULA_Msk & ((value) << CKGR_PLLAR_MULA_Pos)))
+#define CKGR_PLLAR_ONE (0x1u << 29)
+
+#define PMC_MCKR_PRES_CLK_2 (0x1u << 4)
+#define PMC_MCKR_CSS_PLLA_CLK (0x2u << 0)
+#define PMC_MCKR_MDIV_PCK_DIV2 (0x1u << 8)
+
+#define RTEMS_MCK_FREQUENCY_FOR_SYSTICK 75000000U
+
 #define MEGA_HZ 1000000u
 #ifndef MAIN_CRYSTAL_OSCILLATOR_FREQUENCY
 #define MAIN_CRYSTAL_OSCILLATOR_FREQUENCY (12 * MEGA_HZ)
 #endif
 
+// structure used to overwrite RTEMS clock settings,
+// it is necessary because of PLLA and master clock reconfiguration during init
+// see https://docs.rtems.org/docs/6.1/user/bsps/arm/atsam.html
+const struct atsam_clock_config atsam_clock_config = {
+  .pllar_init = (CKGR_PLLAR_ONE | CKGR_PLLAR_MULA(0x28U) |
```

Comment:
Can we untangle these magic numbers? How are they derived?

Reply:
Added defines, they are consistant with pmc configuration for CAN